app/components/trainees/record_details/view.rb

```diff
@@ -39,6 +39,21 @@ def trn_row
         end
       end
 
+      def trainee_status_row
+        {
+          key: "Trainee status",
+          value: render(StatusTag::View.new(trainee: trainee, css_classes: "govuk-!-margin-bottom-2")) + tag.br + trainee_status_date,
```

Comment:
Does this mean there's an extra `<br>` for all trainee statuses that don't have a date?

Reply:
Yes, is there something else that you would suggest?

Reply:
As discussed this won't be an issue once we're only rendering the row for withdrawn and deferred trainees - there'll _always_ be a date.